app/scripts/services/stats.js

```diff
@@ -33,6 +33,7 @@ angular.module('stats', ['sql', 'health', 'tableinfo', 'nodeinfo'])
           setReachability(true);
         } else {
           data.version = null;
+          var storedURI = localStorage.getItem("crate.base_uri");
```

Comment:
baseURI will return the current host if there is nothing stored in the localStorage. It doesn't allow you to check if there is something stored there or not .. unless you want to move this log message to the BaseURI factory

```
  .factory('baseURI', function($location, $log) {
    return function baseURI(path) {
      var basePath = localStorage.getItem("crate.base_uri");
      if (!basePath) {
        $log.warn('...'); // <----
        var pluginPath = '/_plugin/crate-admin/';
        basePath = $location.protocol() + "://" +
          $location.host() + ":" +
          $location.port() +
          window.location.pathname.replace(pluginPath, '');
      }
      // remove trailing slash from base path and append path
      return basePath.replace(/\/+$/, '') + path;
    };
  })
```